pkg/nfd-master/nfd-master.go

```diff
@@ -510,6 +495,26 @@ func (m *nfdMaster) filterFeatureLabels(labels Labels) (Labels, ExtendedResource
 	return outLabels, extendedResources
 }
 
+func (m *nfdMaster) filterFeatureLabel(name string) error {
```

Comment:
I would prefer to name the argument `label` instead of `name`, but that's not a big deal.